From 98f8fc94cea9a96ac403053f4ceb9221e4f28779 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Tue, 6 Dec 2016 14:08:20 -0600
Subject: [PATCH] fix symlink from gvimdiff and gview

---
 src/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/Makefile b/src/Makefile
index 7a65677..a30f940 100644
--- a/src/Makefile
+++ b/src/Makefile
@@ -2554,7 +2554,7 @@ $(DEST_BIN)/$(GVIMTARGET):
 	cd $(DEST_BIN); ln -s $(VIMTARGET) $(GVIMTARGET)
 
 $(DEST_BIN)/$(GVIEWTARGET):
-	cd $(DEST_BIN); ln -s $(VIMTARGET) $(GVIEWTARGET)
+	cd $(DEST_BIN); ln -s g$(VIMTARGET) $(GVIEWTARGET)
 
 $(DEST_BIN)/$(RVIMTARGET):
 	cd $(DEST_BIN); ln -s $(VIMTARGET) $(RVIMTARGET)
@@ -2572,7 +2572,7 @@ $(DEST_BIN)/$(VIMDIFFTARGET):
 	cd $(DEST_BIN); ln -s $(VIMTARGET) $(VIMDIFFTARGET)
 
 $(DEST_BIN)/$(GVIMDIFFTARGET):
-	cd $(DEST_BIN); ln -s $(VIMTARGET) $(GVIMDIFFTARGET)
+	cd $(DEST_BIN); ln -s g$(VIMTARGET) $(GVIMDIFFTARGET)
 
 $(DEST_BIN)/$(EVIMTARGET):
 	cd $(DEST_BIN); ln -s $(VIMTARGET) $(EVIMTARGET)
-- 
2.11.0

