From 98f8fc94cea9a96ac403053f4ceb9221e4f28779 Mon Sep 17 00:00:00 2001
From: Miguel Bernal Marin <miguel.bernal.marin@linux.intel.com>
Date: Tue, 6 Dec 2016 14:08:20 -0600
Subject: [PATCH] fix symlink from gvimdiff and gview

---
 src/Makefile | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- vim-9.0.1505/src/Makefile~	2023-05-02 15:25:47.000000000 +0000
+++ vim-9.0.1505/src/Makefile	2023-05-02 16:48:36.001082268 +0000
@@ -2688,7 +2688,7 @@
 	cd $(DEST_BIN); ln -s $(VIMTARGET) $(VIEWTARGET)
 
 $(DEST_BIN)/$(GVIMTARGET): $(DEST_BIN)
-	cd $(DEST_BIN); ln -s $(VIMTARGET) $(GVIMTARGET)
+	cd $(DEST_BIN); ln -s g$(VIMTARGET) $(GVIMTARGET)
 
 $(DEST_BIN)/$(GVIEWTARGET): $(DEST_BIN)
 	cd $(DEST_BIN); ln -s $(VIMTARGET) $(GVIEWTARGET)
@@ -2709,7 +2709,7 @@
 	cd $(DEST_BIN); ln -s $(VIMTARGET) $(VIMDIFFTARGET)
 
 $(DEST_BIN)/$(GVIMDIFFTARGET): $(DEST_BIN)
-	cd $(DEST_BIN); ln -s $(VIMTARGET) $(GVIMDIFFTARGET)
+	cd $(DEST_BIN); ln -s g$(VIMTARGET) $(GVIMDIFFTARGET)
 
 $(DEST_BIN)/$(EVIMTARGET): $(DEST_BIN)
 	cd $(DEST_BIN); ln -s $(VIMTARGET) $(EVIMTARGET)
